src/main/java/com/kodilla/ecommercee/CartController/CartController.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.ecommercee.CartController;
+
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartDto;
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartItemDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("v1/carts")
+@RequiredArgsConstructor
+public class CartController {
+
+    @GetMapping("/cart")
+    public List<CartItemDto> getCart() {
+        CartItemDto cartItem1 = new CartItemDto(1L, "Kurtka", "Kurtka elegancka jesienna", 750, 2);
+        CartItemDto cartItem2 = new CartItemDto(2L, "Kurtka", "Kurtka sportowa jesienna", 850, 2);
+        CartItemDto cartItem3 = new CartItemDto(3L, "Kurtka", "Kurtka zimowa narciarska", 1200, 5);
+        List<CartItemDto> cart = new ArrayList<>();
+        cart.add(cartItem1);
+        cart.add(cartItem2);
+        cart.add(cartItem3);
+
+        return cart;
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> addCartItem(@RequestBody CartItemDto cartItemDto) {
+        return ResponseEntity.ok().build();
+    }
+
+    @DeleteMapping(value = {"cartItemId"})
+    public ResponseEntity<Void> removeCartItem(@PathVariable Long cartItemId) {
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/order")
```

Comment:
I'm assuming you will need @PostMapping here. Instead of @GetMapping I would do it like this:

@PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
public ResponseEntity<Void> createOrder(@RequestParam long cartId, @RequestParam long userId) {
return (ResponseEntity.ok().build();
}

For now it can be Void. In the future we can create Dto for it or leave it as Void (depends on requirements).
For @RequestParam Your Postman call will look like this:
http://localhost:8080/v1/carts?cartId=3&userId=2

Reply:
Please refer to commit:
[JDP230102-3_CartController_initial_update_1](https://github.com/interceptor2048/project-jdp-2301-02/pull/8/commits/53a0e8698bcec7cf6306df4f688ec9e5b25f1c34)

    @PostMapping(value = "/order")
    public ResponseEntity<CartItemDto> createOrder() {
        //Method will return OrderDto when it is ready//
        //Get current cart from repository//
        //Map current cart to OrderDto//
        return ResponseEntity.ok().build();
    }